core/account.py

```diff
@@ -86,6 +86,29 @@ def get_bulk_accounts(
         return {"error": "No matching account found"}, 404
 
 
+# old roles, we should remove these once all old roles are removed in azure
+_DEPRECATED_ROLES = {"LEAD_ASSESSOR", "ASSESSOR", "COMMENTER"}
+
+_VALID_ROLES = {
```

Comment:
could we use the az_role_map in config here instead?

Reply:
Ah! yes! good eye

Reply:
Actually, they are slightly different things - `azure_ad_role_map` is a mapping from the azure role to our role. 

This map is a mapping from legacy roles (LEAD_ASSESSOR, ASSESSOR & COMMENTER) to the new roles, but also includes a 1:1 mapping for all other roles, I've reverted this change!

Reply:
okay fair dos, thanks for looking